### PR TITLE
[feature](Cloud) Alter be to fetch vault info after creating storage vault

### PR DESCRIPTION
**Pull request by ByteYue**
## Proposed changes

Issue Number: close #xxx

Make BE eagerly fetches the vault info right after FE receives the create vault resp from MS.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by ByteYue** [comment]
run feut

File: be/src/service/internal_service.cpp [hunk 1]
```diff
@@ -2054,4 +2055,11 @@ void PInternalService::get_wal_queue_size(google::protobuf::RpcController* contr
     }
 }
 
+void PInternalServiceImpl::alter_vault_sync(google::protobuf::RpcController* controller,
+                                            const doris::PAlterVaultSyncRequest* request,
+                                            PAlterVaultSyncResponse* response,
+                                            google::protobuf::Closure* done) {
+    _engine.to_cloud().sync_storage_vault();
```
Comment (platoneko) [inline be/src/service/internal_service.cpp:2062]: Implement this method in `CloudBackendService`, `to_cloud()` here will check failed

**Review by platoneko — CHANGES_REQUESTED** [review]

**Comment by ByteYue** [comment]
run buildall

**Review by platoneko — APPROVED** [review]

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**